src/routes/[topicSlug]/[tableSlug]/[categorySlug].svelte

```diff
@@ -22,36 +22,39 @@
     getCategoryBySlug,
     populatesSelectedData,
     selectedData,
+    fetchSelectedDataForGeographies,
   } from "../../../model/censusdata/censusdata";
-  import {
-    updateHoveredGeography,
-    updateSelectedGeography,
-    getLadName,
-    selectedGeography,
-  } from "../../../model/geography/geography";
+  import GeodataApiDataService from "../../../model/censusdata/services/geodataApiDataService";
+  import { updateHoveredGeography, updateSelectedGeography, getLadName } from "../../../model/geography/geography";
   import config from "../../../config";
   import TileSet from "../../../ui/map/TileSet.svelte";
   import InteractiveLayer from "../../../ui/map/InteractiveLayer.svelte";
   import BoundaryLayer from "../../../ui/map/BoundaryLayer.svelte";
   import DataLayer from "../../../ui/map/DataLayer.svelte";
   import { appIsInitialised } from "../../../model/appstate";
-  import { isNotEmpty } from "../../../utils";
+  import { isNotEmpty, categoryIDToDBColumn, categoryIDToDBTotalsColumn } from "../../../utils";
 
   import { page } from "$app/stores";
   import { onMount } from "svelte";
 
   let { topicSlug, tableSlug, categorySlug } = $page.params;
   let category = null;
   let table = null;
+  let populateCensusTable = { categories: [] };
 
-  let locationId = null;
   let locationName = "";
-  locationId = $page.query.get("location");
+
+  const locationId = $page.query.get("location");
```

Comment:
is locationId and geocode the same bit of information? Do you need both?

Reply:
We originally used one variable for both, with locationId defaulting to the England and Wales code in the absence of a location query parameter.  But the locationId is handed into a few components and it caused problems because they were looking for E&W in the LAD list, and the E&W code was appearing in the url on screen which we don't want.  All problems that could be fixed but maybe out of scope for this ticket, and maybe would be useful to have a conversation with Lynsey first about how we want to handle E&W as a selection in the app.

Reply:
👍